src/textual/app.py

```diff
@@ -4325,10 +4355,12 @@ def suspend(self) -> Iterator[None]:
             # app, and we don't want to have the driver auto-restart
             # application mode when the application comes back to the
             # foreground, in this context.
+            # fmt: off
```

Comment:
Ruff formats this code into parentheses, and you can't use parentheses in context managers on 3.8.

Reply:
Fixed by adding:

```toml
[tool.ruff]
target-version = "py38"
```

To `pyproject.toml`.